### PR TITLE
added info logs

### PR DESCRIPTION
**Pull request by singhsangeet7**
### What issues does this PR fix or reference?
Fixes:

### Previous Behavior
Remove this section if not relevant

### New Behavior
Remove this section if not relevant

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**

- [ ] Docs
- [ ] Changelog - https://docs.saltstack.com/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltstack.com/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

### CODE REVIEW

**Comment by s0undt3ch** [comment]
Why?

File: salt/auth/file.py [hunk 1]
```diff
@@ -248,7 +248,7 @@ def _htfile(username, password, **kwargs):
     """
 
     filetype = kwargs.get("filetype", "htpasswd").lower()
-
+    log.info("filetype is {}".format(filetype))
```
Comment (Ch3LL) [inline salt/auth/file.py:251]: can you elaborate why you are adding these log statements?

Comment (Ch3LL) [inline salt/auth/file.py:251]: can you elaborate why you are adding these log statements?

**Review by Ch3LL — CHANGES_REQUESTED** [review]

**Comment by dwoz** [comment]
Closing this due to inactivity. Anyone should feel free to re-open it if they want to see it through to the end in one release cycle.

**Pull request closed without merging**